.mk/ocp.mk

```diff
@@ -25,9 +25,17 @@ ocp-deploy: deploy-all ocp-expose-all ## OCP deploy (loki, grafana, example-cr a
 ocp-undeploy: undeploy-all  ## OCP cleanup
 
 .PHONY: ocp-run
-ocp-run: ocp-undeploy ocp-deploy   ## OCP-deploy + run the operator locally
+ocp-run: ocp-undeploy ocp-deploy ocp-deploy-flp  ## OCP-deploy + run the operator locally
+
+.PHONY: ocp-deploy-flp
+ocp-deploy-flp: ## run flp from the operator
```

Comment:
same here, I'm puzzled with the name, as it's running the operator (which itself will start FLP indirectly, ok, but still the actual action is to start the operator.

Reply:
Changed to ocp-operator-deploy.